Extension/src/Debugger/configurationProvider.ts

```diff
@@ -38,6 +38,8 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
     private assetProvider: IConfigurationAssetProvider;
     // Keep a list of tasks detected by cppBuildTaskProvider.
     private static detectedBuildTasks: CppBuildTask[] = [];
```

Comment:
Is this 3rd array still needed?

Reply:
Yeah, that's the one that is actually used. The c or cpp cache gets copied into it depending on the current file's type.